test/transport/test_serial.py

```diff
@@ -96,7 +96,7 @@ async def test_write_force_poll(self):
         """Test write with poll."""
         SerialTransport.force_poll = True
         transport, protocol = await create_serial_connection(
-            asyncio.get_running_loop(), mock.Mock, url="dummy"
+            asyncio.get_running_loop(), mock.Mock, None, url="dummy"
```

Comment:
So there are NO test at all of the rs485 settings.

Transport have 100% coverage, please ensure you still keep that.

Reply:
Sure thing - when you say coverage are you talking about LOC coverage?

Reply:
I am talking about coverage. Please see the CI run.